query-engine/connectors/sql-query-connector/src/join_utils.rs

```diff
@@ -93,62 +115,72 @@ fn compute_aggr_join_one2m(
 fn compute_aggr_join_m2m(
```

Comment:
m2m joins are rendered with a join and a subselect.

The subselect was made on the parent model instead of the child model, preventing filters from easily being integrated since the child table on which filters are supposed to be executed wasn't selected.

I refactored the subselect so that it fetches the child table. Filters can now integrate nicely.